### ISSUE TITLE
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputRadio.Name

### ISSUE THREAD

**Issue by ttnguyen2** [GSA/touchpoints#1105]
@DaveSkender
@ryanwoldatwork 
We're running pa11y accessibility tests and getting hits on this specific rule on our modal form embedded as a tab.
 Error: "This radioinput element does not have a name available to an accessibility API. 
 Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.",
 

input value="0" id="star0" checked="" type="radio" name="rating" hidden="" style="display: none"

**Comment by qjac** [comment]
+1. We are also getting this error.

**Comment by ryanwoldatwork** [comment]
This has been resolved, as the 0 value option now has an idea with an aria-label.
